examples/ecmascript/es5.ohm

```diff
@@ -273,15 +273,12 @@ ES5 {
                     | "(" Expression ")"  -- parenExpr
 
   // @returns an ArrayExpr AST
-  ArrayLiteral = "[" AssignmentExpressionOrElisionComma* "]"                       -- trailingComma
-               | "[" AssignmentExpressionOrElisionComma* AssignmentExpression "]"  -- noTrailingComma
-  AssignmentExpressionOrElisionComma = (AssignmentExpression | Elision) ","
-  Elision =
+  ArrayLiteral = "[" ListOf<AssignmentExpressionOrElision, ","> "]"
+  AssignmentExpressionOrElision = AssignmentExpression
+                                |                       -- elision
```

Comment:
In this case it's spec compliant: You can have as many `,` in `[]` as you like.
`[]` = empty array, length 0
`[,]` = empty array, length 1
`[,,]` = empty array, length 2, etc.